### ISSUE TITLE
{ op='copy' } is not standard compliant

### ISSUE THREAD

**Issue by alex94puchades** [dharmafly/jsonpatch.js#21]
According to the [rfc](https://tools.ietf.org/html/rfc6902#section-4.5), the following code should give an error:
```js
jsonpatch.apply_patch({}, { op: 'copy', from: '/invalid', path: '/dest' })
```
However, it returns ```{ dest: undefined }```

**Comment by almost** [comment]
This definitely looks like a bug. I can try and get a fix for this soonish. Pull Requests very welcome though :)

**Comment by alex94puchades** [comment]
Sorry, real busy right now. It's not a critical bug for me, so if it can
wait till September I'll take a look (should be easy though).

--
Álex Puchades

**Comment by almost** [comment]
This is now fixed and published to NPM (version 2.0.0). Thanks to @agraboso!
